src/orange/orangeinp/IntersectRegion.hh

```diff
@@ -254,7 +342,8 @@ class GenPrism final : public IntersectRegionInterface
                               TrapFace const& lo,
                               TrapFace const& hi);
 
-    // Construct from half Z height and 4 vertices for top and bottom planes
+    // Construct from half Z height and 4 vertices for top and bottom
+    // planes
```

Comment:
Maybe your editor somehow has comment lengths set to less than 80? Can you try to revert these changes?

Reply:
Sure. I am using off-the-shelf clang-format, so it should be respecting whatever is in `.clang-format`. I am really not sure what is going on and why it only affects these lines (if it was a 76-character problem there would probably be a lot more changes). Frustratingly, this is exactly the kind of tedious work clang-format is supposed to be preventing.

Reply:
This isn't happening from inside vscode?

Reply:
This MR was done completely in `vim`. My `vimrc` is nearly empty.

Reply:
Weird... no idea why it would reformat those then, especially if you didn't touch them. Do you use the git-clang-format commit hook? (Installed via scripts/dev/install-commit-hooks.sh)